core/amber/src/main/resources/application.conf

```diff
@@ -91,3 +91,7 @@ region-plan-generator {
     enable-cost-based-region-plan-generator = false
     use-global-search = false
 }
+
+language-server{
+    default = "pyright"
+}
```

Comment:
```suggestion
language-server {
    provider = "pyright" # valid options: ["pyright", "pylsp"]
    port = 3000
}
```

Reply:
1. don't name a config as `default`. give it a name and give it a default value. you can also provide other valid options in comments.
2. I suggest make the port configurable. (you probably need to change both frontend and backend to read the config).